admin/src/components/Nfts.tsx

```diff
@@ -170,7 +180,13 @@ const NftAside = ({ ...props }) => {
                         {attrKey[0].toUpperCase() + attrKey.replace('_', ' ').slice(1)}
                       </Typography>
                       <Typography variant="body2" style={{ fontFamily: 'Poppins Medium', marginLeft: '1em', marginBottom: '1em', marginTop: '0.5em' }}>
-                        {props.record.attributes[attrKey]}
+                        {
+                          attrKey === 'launch_at' ?
```

Comment:
Think this shouldn't be matched on the attribute key, it can match on attribute kind `date`